block-storage/src/main/scala/coop/rchain/blockstorage/dag/BlockDagRepresentationSyntax.scala

```diff
@@ -12,60 +14,93 @@ import coop.rchain.models.syntax._
 import coop.rchain.shared.syntax._
 import fs2.Stream
 
-trait BlockDagRepresentationSyntax {
-  implicit final def blockStorageSyntaxBlockDagRepresentation[F[_]](
-      dag: BlockDagRepresentation[F]
-  ): BlockDagRepresentationOps[F] = new BlockDagRepresentationOps[F](dag)
+import scala.collection.immutable.SortedMap
+
+trait DagRepresentationSyntax {
```

Comment:
DagRepresentationSyntax is created to be able to use across the code the same API that BlockDagRepreentation[F[_]] had to minimise changes in the logic code. Methods require BlockDagStorage where access to metadata is needed.